setup.py

```diff
@@ -29,7 +18,7 @@
               "infinity"],
     license="Apache",
     url="https://github.com/AlekseyKorshuk/optimum-transformers",
-    install_requires=requirements,
+    install_requires=open("requirements.txt", "r", encoding="utf-8").read().splitlines(),
```

Comment:
Ok I have added the `open` function directly to the setup. Is it resolving the file problem ?

If not I saw some changes to bring to the ci pipeline to make it work with a requirements.txt file, or we could directly declare dependencies in the setup.py file and remove the requirements.txt file.

What do you think ?

Reply:
Lets give it a try